pkg/config/application.go

```diff
@@ -564,15 +567,18 @@ type Attachment struct {
 	Targets []string `json:"targets"`
 }
 
-func (e *Attachment) Validate() error {
-	for k, v := range e.Sources {
+func (a *Attachment) Validate() error {
+	for k, v := range a.Sources {
 		if k == "" {
 			return fmt.Errorf("key field in sources must not be empty")
 		}
 		if v == "" {
 			return fmt.Errorf("value field in sources must not be empty")
 		}
 	}
+	if len(a.Targets) == 0 {
+		return fmt.Errorf("no target files to be embedded")
```

Comment:
[IMO] How about "attachment's targets must not be empty"?

Reply:
That's because I think users can more easily find the cause&solution if "attachment's targets" is in the message.